apps/dot-voting/app/components/VoteDetails/index.js

```diff
@@ -121,8 +163,10 @@ const VoteDetails = ({ app, vote, userAccount, onVote }) => {
 VoteDetails.propTypes = {
   app: PropTypes.object.isRequired,
   userAccount: PropTypes.string.isRequired,
+  tokenContract: PropTypes.object.isRequired,
   vote: PropTypes.object.isRequired,
   onVote: PropTypes.func.isRequired,
+  setCurrentVoteId: PropTypes.func.isRequired,
```

Comment:
We did not start passing this in, or using it anywhere.

Reply:
Removed.